jobs/openshift/cluster/integreatly/test/Jenkinsfile

```diff
@@ -10,9 +10,6 @@ def verifyParameters(testOptions) {
 
     testOptions.gitUrl = userInput.installationGitUrl
     testOptions.gitBranch = userInput.installationGitBranch
-    testOptions.clusterName = userInput.installationGitBranch
-    testOptions.openshiftMasterUrl = "https://${userInput.installationGitBranch}.${testOptions.domainName}"
-    verifyParameters(testOptions)
```

Comment:
This should be added back "verifyParameters". It's called from within itself to ensure that whatever the user inputs is also verified. It will just constantly ask for input until such time as the user has entered valid input. As you have it now, someone could just enter nothing and press continue and it would carry on but with invalid inputs and fail at some point later on.

Reply:
OK I didn't understand that. I'll put it back.